src/perl/vdotest/VDOTest.pm

```diff
@@ -148,6 +148,8 @@ our %PROPERTIES
      userBinaryDir           => undef,
      # @ple Use the dmlinux src rpm for testing.
      useUpstreamKernel       => 0,
+     # @ple Use the installed kernel module, dm-vdo
+     useLinuxNextModule      => 0,
     );
```

Comment:
useLinuxNextModule is a misleading name; the in-kernel dm-vdo module does not have to come from linux-next. Instead it will generally be the normal upstream module.
I really think useUpstreamModule is a more correct name for this property. You will probably also want to rename the existing useUpstreamKernel property to avoid confusion; it should properly be called something like useDmLinuxModule.

Reply:
This is still true.

Renaming useUpstreamKernel is probably best done as a separate commit, but it should still be in this PR.